pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -173,6 +173,24 @@ def test_is_iequivalent(self):
         G3 = BayesianModel([('W', 'V'), ('W', 'X'), ('Y', 'X'), ('Z', 'Y')])
         self.assertFalse(G3.is_iequivalent(G2))
 
+    def test_copy(self):
+        model_copy = self.G1.copy()
+        self.assertEqual(sorted(self.G1.nodes()), sorted(model_copy.nodes()))
+        self.assertEqual(sorted(self.G1.edges()), sorted(model_copy.edges()))
+        self.assertNotEqual(id(self.G1.get_cpds('diff')),
+                            id(model_copy.get_cpds('diff')))
```

Comment:
@ankurankan I think more tests for the copy method can be added. Tests to ensure that changes in original do not get reflected in the copy and vice versa.